helper_functions.c

```diff
@@ -114,7 +114,7 @@ static inline void cpuid (unsigned int info, unsigned int *eax, unsigned int *eb
     if (edx) *edx = _edx;
 }
 
-static inline void  get_vendor (char *vendor_string)
+static inline void  get_vendor (char vendor_string[static 13])
```

Comment:
Whoops. looks like I said that I didn't do this in my commit message, but I still did. Behavior is nonetheless correct.